### PR TITLE
Update front facing videos

### PR DESCRIPTION
**Pull request by hidmic**
### Proposed changes

Precisely what the title says. This patch updates both the repository README and the documentation index page to show a slightly more enticing video of Beluga in action.

#### Type of change

- [ ] 🐛 Bugfix (change which fixes an issue)
- [ ] 🚀 Feature (change which adds functionality)
- [x] 📚 Documentation (change which fixes or extends documentation)

### Checklist

- [ ] Lint and unit tests (if any) pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All commits have been signed for [DCO](https://developercertificate.org/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ Beluga is an extensible C++17 library with a ground-up implementation of the Mon
 - Emphasis on the prevention of regressions and facilitation of code improvements through test coverage.
 - Semi-automated benchmarks that can be used to validate different configurations.
 
+https://github.com/user-attachments/assets/59bca7ea-fe78-4460-b058-96a5c75d09ec
```
Comment (hidmic) [inline README.md:26]: @nahueespinosa is this the correct way to make the video show up among `Ekumen-OS/beluga/assets`?

Comment (nahueespinosa) [reply README.md:26]: Not sure what you mean, but this looks fine by me.

**Review by nahueespinosa — APPROVED** [review]
LGTM!

**Comment by hidmic** [comment]
Skipping CI as this does not affect code.

**Pull request merged**